content/en/docs/ambient/architecture/index.md

```diff
@@ -9,15 +9,66 @@ owner: istio/wg-networking-maintainers
 test: n/a
 ---
 
+## Ambient APIs
+
+### Labels
+
+You can use the following labels to enroll your namespace to ambient, enroll your resource to use a waypoint and specify, the capture scope of your waypoint.
+
+|  Name  | Feature Status | Scope | Description |
+| --- | --- | --- | --- | --- |
+| `istio.io/dataplane-mode` | Beta | `Namespace` |  Specifies the data plane mode, valid value: `ambient`. |
+| `istio.io/use-waypoint` | Beta | `Namespace` or `Service` or `Pod` | Enrolls your resource to use a given waypoint, valid values: `{waypoint-name}`, `{namespace}/{waypoint-name}`, or `#none` |
+| `istio.io/waypoint-for` | Alpha | `Gateway` | Specifies the waypoint's capture scope, valid value: `service` or `none` or `workload` or `all`. This label is optional and the default value is `service`. |
+
+In order for your `istio.io/use-waypoint` label value to be effective, you have to ensure the used waypoint is configured with the corresponding capture scope which is `service` by default. For example, when you label a pod to use a specific waypoint via the `istio.io/use-waypoint` label, the waypoint's capture scope should be `workload` or `all`.
+
+### Policy attachment to waypoints
+
+You can attach Layer 7 policies (such as `AuthorizationPolicy`, `RequestAuthentication`, `Telemetry`, `WasmPlugin`, etc) to your waypoint using `targetRef`.
+
+- To attach the entire waypoint, set `Gateway` as the `targetRef` value. The example below shows how to attach the `viewer` policy
+to the waypoint named `waypoint` for the `default` namespace:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: viewer
+      namespace: default
+    spec:
+      targetRefs:
+      - kind: Gateway
+        group: gateway.networking.k8s.io
+        name: waypoint
+    {{< /text >}}
+
+- To attach a given service within the waypoint, set `Service` as the `targetRef` value. The example below shows how to attach
+the `productpage-viewer` policy to the `productpage` service in the `default` namespace:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: productpage-viewer
+      namespace: default
+    spec:
+      targetRefs:
+      - kind: Service
+        group: ""
+        name: productpage
+    {{< /text >}}
+
 ## Traffic routing
 
 In {{< gloss "ambient" >}}ambient mode{{< /gloss >}}, workloads can fall into 3 categories:
```

Comment:
It feels odd to talk about traffic capture _after_ we talk about L7 policy enforcement via waypoints, which requires traffic capture to work.

Feels like this whole section needs to be at the top.

Reply:
Maybe :) @craigbox said he would reorganize things once contents are in.  It is possible we may want Ambient APIs to have its own page.  I'll defer this organization as a follow up.